### PR TITLE
More restructuring 

### PR DESCRIPTION
**Pull request by Jimbo4350**
https://github.com/input-output-hk/cardano-node/issues/84

### CODE REVIEW

File: cardano-node/cardano-node.cabal [hunk 1]
```diff
@@ -24,24 +24,24 @@ library
                        -- Conversion
                        Cardano.Node.Configuration.Conversion
                        Cardano.Node.Configuration.Partial
+                       Cardano.Node.Configuration.Topology
                        Cardano.Node.Configuration.Types
                        Cardano.Node.Configuration.Presets
```
Comment (intricate) [inline cardano-node/cardano-node.cabal]: This should probably come before `Cardano.Node.Configuration.Topology`

File: cardano-node/src/Cardano/Node/TUI/GitRev.hs [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 {-# OPTIONS_GHC -Wno-simplifiable-class-constraints #-}
 
-module Cardano.Node.GitRev (
+module Cardano.Node.TUI.GitRev (
```
Comment (intricate) [inline cardano-node/src/Cardano/Node/TUI/GitRev.hs]: What does TUI stand for?

Comment (Jimbo4350) [reply cardano-node/src/Cardano/Node/TUI/GitRev.hs]: https://en.wikipedia.org/wiki/Text-based_user_interface

Comment (intricate) [reply cardano-node/src/Cardano/Node/TUI/GitRev.hs]: Ah, okay. I wonder if `TerminalUI` or `TextUI` might be more clear. Unless `TUI` is pretty common and people generally understand it (@deepfire, what do you think?). Perhaps it's just me who hasn't seen this acronym before.

Comment (intricate) [reply cardano-node/src/Cardano/Node/TUI/GitRev.hs]: Either way, I wouldn't block the PR on this

Comment (deepfire) [reply cardano-node/src/Cardano/Node/TUI/GitRev.hs]: I think `TUI` is good.

**Pull request merged**